pkg/kubectl/cmd/autoscale.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"k8s.io/kubernetes/pkg/kubectl"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/util/errors"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	autoscaleLong = `Automatically increase or decrease number of pods deployed within the system as needed.
+
+<add more description>.`
+
+	autoscaleExample = `  # Auto scale a replication controller "foo":
+  $ kubectl autoscale rc foo --min=3 --max=10
+
+<add more examples>`
+)
+
+// TODO: support autoscale for deployments
+func NewCmdAutoscale(f *cmdutil.Factory, out io.Writer) *cobra.Command {
+	filenames := []string{}
+	cmd := &cobra.Command{
+		Use:     "autoscale (-f FILENAME | TYPE NAME | TYPE/NAME)",
```

Comment:
We should be consistent with all other comments. For example:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/expose.go#L64

Also, square brackets usually imply that the flag is optional. In this case, one of these arguments is required.